order/views.py

```diff
@@ -89,9 +81,9 @@ def dydx_order(self, request):
             orders = vars(orders)
             orders_data = orders["data"]["account"]["openPositions"]
             result["message"] = orders_data
-            if not orders_data == {}:
-                return Response(result, status.HTTP_200_OK)
-            result["message"] = "No openPositions on dydx"
+            if orders_data is {}:
+                if orders is None:
+                    raise Exception("No open position found on dydx")
```

Comment:
if orders_data is empty, then technically we have no orders open on dydx right ? 

If yes, then why are we checking **if orders is None**. Cause orders doesn't necessarily have to always be none for orders_data to be none as well.

Reply:
right  i have change it